index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Ivan Martinez - Portfolio</title>
+   <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+   <header>
+      <nav>
+          <img class="img1" src="resources/image_geometry_1.svg" alt="Mosaic above">
+          <img src="resources/ic_menu.svg" alt="Menu">
```

Comment:
Kindly wrap the hamburger in the `<nav>` tag 😃

The nav tag is for semantics reasons. Later in the project, you will add a navigation menu that opens when you click the hamburger.